.github/file-filter.yml

```diff
@@ -28,8 +28,10 @@ tee_src: &tee_src
   - 'tee-worker/build.Dockerfile'
   - 'tee-worker/enclave-runtime/**'
   - 'tee-worker/docker/**'
+  - 'tee-worker/client-api/**'
```

Comment:
Is it intended? Do we need to rebuild the tee image for any file change under this folder?

Reply:
I have temporarily removed it here. It was added earlier to fix an issue with CI, but now it may not be necessary.